src/PolyhedralGeometry/helpers.jl

```diff
@@ -19,6 +19,43 @@ Base.convert(::Type{Polymake.Integer}, x::fmpz) = Polymake.Integer(BigInt(x))
 Base.convert(::Type{Polymake.Rational}, x::fmpz) = Polymake.Rational(convert(Polymake.Integer, x), convert(Polymake.Integer, 1))
 Base.convert(::Type{Polymake.Rational}, x::fmpq) = Polymake.Rational(convert(Polymake.Integer, numerator(x)), convert(Polymake.Integer, denominator(x)))
 
+const nf_scalar = Union{nf_elem, fmpq}
+
+# export nf_scalar
+
+Base.zero(::Type{nf_scalar}) = fmpq()
+# Base.one(::Type{nf_scalar}) = fmpq(1)
+
+Base.convert(::Type{nf_scalar}, x::Number) = convert(fmpq, x)
+Base.convert(::Type{nf_scalar}, x::nf_elem) = x
+
+nf_scalar(x::Union{Number, nf_elem}) = convert(nf_scalar, x)
+
+function Base.convert(::Type{Polymake.QuadraticExtension{Polymake.Rational}}, x::nf_elem)
+    p = defining_polynomial(parent(x))
+    if (length(p) != 3 && coeff(p, 1) != 0) || sign(coeff(p, 2)) == sign(coeff(p, 0))
```

Comment:
```suggestion
    if length(p) != 3 || coeff(p, 1) != 0 || sign(coeff(p, 2)) == sign(coeff(p, 0))
```
Or maybe we can use something like this:
```suggestion
    isq = Hecke.isquadratic_type(parent(x))
    if isq[1] == true && isq[2] > 0
```